.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "submodules/account-abstraction"]
 	path = submodules/account-abstraction
 	url = https://github.com/eth-infinitism/account-abstraction.git
-	branch = releases/v0.7
+	branch = callValidateUserOp-catch-revert
```

Comment:
This is temporary, no?

Reply:
yes, though the new value will be "develop", until we complete releases/v0.8